Core/Src/mainTask.c

```diff
@@ -40,13 +41,12 @@ static bool initialized = false;
 static uint32_t initRetries = 5;
 static uint32_t lastUpdateMain = 0;
 
-bool balancingEnabled = false;
+bool balancingEnabled = true;
```

Comment:
Do you want to leave balancing enabled on main?

Reply:
Probably not